SBMLDiagrams/visualizeSBML.py

```diff
@@ -257,23 +258,24 @@ def hex_to_rgb(value):
 
                         for j in range (0, info.getNumStyles()):
                             style = info.getStyle(j)
-                            group = style.getGroup()
+                            group = color_style.getGroup()
```

Comment:
This should be "style" instead of "color_style". It is from libsbml library.

Reply:
fix it